src/main/java/edu/ucsb/nceas/mdqengine/scheduler/MonitorJob.java

```diff
@@ -0,0 +1,314 @@
+ */
+@DisallowConcurrentExecution
+public class MonitorJob implements Job {
+
+    private Log log = LogFactory.getLog(RequestReportJob.class);
+    private Controller controller = null;
+
+    /**
+     * Default constructor for the MonitorJob class.
+     * Initializes the MonitorJob object by getting an instance of the Controller,
+     * which is a singleton.
+     */
+    public MonitorJob() {
+        this.controller = Controller.getInstance();
+    }
+
+    /**
+     * Execute method for the MonitorJob.
+     *
+     * @param context JobExecutionContext object with environment information
+     * @throws JobExecutionException
+     */
+    public void execute(JobExecutionContext context) throws JobExecutionException {
+
+        MDQStore store = null;
+        List<Run> processing = new ArrayList<Run>();
+        // Get a connection to the database
+
+        try {
+            store = new DatabaseStore();
+        } catch (MetadigStoreException e) {
+            e.printStackTrace();
+            throw new JobExecutionException("Cannot create store, unable to schedule job", e);
+        }
+
+        if (!store.isAvailable()) {
+            try {
+                store.renew();
+            } catch (MetadigStoreException e) {
+                e.printStackTrace();
+                throw new JobExecutionException("Cannot renew store, unable to schedule job", e);
+            }
+        }
+
+        // query database
+        try {
+            processing = store.listInProcessRuns();
+        } catch (MetadigStoreException e) {
+            e.printStackTrace();
+        }
+
+        // request job via rabbitMQ
+        for (Run run : processing) {
+            log.info("Requesting monitor job: " + run.getId() + run.getStatus() + run.getNodeId());
+            Session session = getSession();
+            String pidStr = run.getId();
+            String suiteId = run.getSuiteId();
+            String nodeId = run.getNodeId();
+            // the methods below are pretty clunky
+            InputStream metadata = getMetadata(pidStr, nodeId, session);
+            InputStream sysmeta = getSystemMetadata(pidStr, nodeId, session);
+
+            String localFilePath = null;
+            DateTime requestDateTime = new DateTime(DateTimeZone.forOffsetHours(-7));
+            try {
+                controller.processQualityRequest(nodeId, pidStr, metadata, suiteId,
+                        localFilePath, requestDateTime,
+                        sysmeta);
+            } catch (IOException io) {
+                io.printStackTrace();
+            }
+
+        }
+        store.shutdown();
+    }
+
+    /**
+     * Establishes an authenticated session using the DataONEauthToken in
+     * metadig.properties
+     * 
+     * @return a Session object
+     * 
+     * @throws JobExecutionException if there is an error executing the quartz task
+     */
+    public Session getSession() throws JobExecutionException {
+
+        String DataONEauthToken = null;
```

Comment:
camelcasing on proper nouns is a problem throughout the code I think I'm going to leave it for now

Reply:
your call; whatever you decide to do with the remainder of the letters, though, I would strongly recommend never using an uppercase first letter for a variable. By convention, uppercase first letter implies a class, so elsewhere in the code, `DataONEauthToken.someMethod()` is confusing because it looks like a static method invocation. 

The Google Code Style is helpful (and non-ambiguous) on naming: https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

Reply:
changed it